RadixWallet/Core/DesignSystem/Styles/SecondaryRectangularButtonStyle.swift

```diff
@@ -1,6 +1,7 @@
 // MARK: - SecondaryRectangularButtonStyle
 public struct SecondaryRectangularButtonStyle: ButtonStyle {
 	@Environment(\.controlState) var controlState
+	let backgroundColor: Color
```

Comment:
I'd say we should avoid this, in my mind the background color is a core feature of this particular style. What other color do you need for this button style?

Reply:
I see your point. This is needed for this view, which has a background of `gray3` (and not `gray4`).

<img width="421" alt="Screenshot 2024-04-19 at 13 16 15" src="https://github.com/radixdlt/babylon-wallet-ios/assets/164921079/5fdb07b3-5bda-4fdd-979c-4fb2f07849ae">

I believe it is better having the `backgroundColor` of this style as modifiable as any of its other properties than repeating this logic on a `TertiaryRectangularButtonStyle`.

Reply:
Then I think you should first check if that difference was actually intentional, it could also be a mistake. If it is intentional, and perhaps it's because the view background is not white, then I think there should rather be either a boolean flag like "hasNonWhiteBackground" or a Context enum and parameter that is either something like default, some clever description of this situation here. In short, I think we should strive for "semantic" parameters when possible.